projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php

```diff
@@ -164,6 +164,15 @@ public function is_current_user_subscribed() {
 		return $this->get_token_property( 'blog_sub' ) === 'active';
 	}
 
+	/**
+	 * Returns true if the current authenticated user has a pending subscription to the current site.
+	 *
+	 * @return boolean
+	 */
+	public function is_current_user_pending_subscriber() {
+		return false;
```

Comment:
I believe a user using this service could be pending, no?
Something like
```
$this->get_token_property( 'blog_sub' ) === 'pending'
```

If not, that definitely needs a comment to clarify...

Reply:
this class is abstract, most services that implement this class have their implementation of this method

Reply:
Oh. I missed that. It should be said in the filename I think... Thanks.